src/Microsoft.ML.TimeSeries/SequenceModelerBase.cs

```diff
@@ -75,6 +75,8 @@ private protected SequenceModelerBase()
         /// <summary>
         /// Implementation of <see cref="ICanSaveModel.Save(ModelSaveContext)"/>.
         /// </summary>
-        public abstract void Save(ModelSaveContext ctx);
+        void ICanSaveModel.Save(ModelSaveContext ctx) => SaveModel(ctx);
+
+        private protected abstract void SaveModel(ModelSaveContext ctx);
```

Comment:
>SaveModel [](start = 40, length = 9)

Unlike the other things, I view this as not being `[BestFriend]` appropriate since it is *very* particular to time series. (So I'd keep this one the way it is.)